src/element/textElement.ts

```diff
@@ -888,7 +888,10 @@ const DEFAULT_LINE_HEIGHT = {
 
 export const getDefaultLineHeight = (fontFamily: FontFamilyValues) => {
   if (fontFamily) {
-    return DEFAULT_LINE_HEIGHT[fontFamily];
+    return (
```

Comment:
@CoyoteWaltz could you also add a test for this?

Reply:
test added, pls check